lib/quiet_mash.rb

```diff
@@ -0,0 +1,3 @@
+class QuietMash < Hashie::Mash
+  disable_warnings
+end
```

Comment:
😡 NEW LINE CHARACTERS!!!

Reply:
You're right, and after _just_ having a discussion with someone recently advocating why we should include these 🤦‍♂️